scan_promela/build.rs

```diff
@@ -0,0 +1,64 @@
+use cfgrammar::yacc::YaccKind;
+use lrlex::CTLexerBuilder;
+use std::env;
+
+fn main() {
+    //     env::set_var("RUST_BACKTRACE", "0");
+    /*
+        // Build the lexer and parser with proper error handling
+        let lexer_builder = match CTLexerBuilder::new()
+            .lrpar_config(|ctp| {
+                ctp.yacckind(YaccKind::Grmtools)
+                    .grammar_in_src_dir("spinv4.y")
+                    .map_err(|e| format!("Errore nella configurazione della grammatica: {}", e))
+                    .unwrap()
+            })
+            .lexer_in_src_dir("spinv.l")
+            .map_err(|e| format!("Errore nell'inizializzazione del lexer: {}", e))
+        {
+            Ok(builder) => builder,
+            Err(e) => {
+                eprintln!("{}", e);
+                std::process::exit(1);
+            }
+        };
+    */
+    // Build the lexer and parser with proper error handling
+    let lexer_builder = match CTLexerBuilder::new()
+        .lrpar_config(|ctp| {
+            println!("Inizializzazione di lrpar_config...");
+            let result = ctp
+                .yacckind(YaccKind::Grmtools)
+                .grammar_in_src_dir(r".\main\spinv4.y")
```

Comment:
These paths only work on Windows, not on  Unix-based platforms (Linux, MacOS) because it uses `\` instead of `/`. To produce a cross-platform compatible path,  use `PathBuf` from the standard library instead of a string.